config/fixes/convention-eccodes.yaml

```diff
@@ -347,7 +358,7 @@ convention_name:
           - mslhf # legacy eccodes
           - slhf
           - SLHF # AQUA ERA5
-          - hfls # cmor
+          # - hfls # cmor --> sign wrong
```

Comment:
I am confused, this means we cannot use the convention here?

Reply:
Exactly. There is a range of fluxes which are positive in cmor (eg hfls)  and would have an opposite sign convention in grib. Currently all these are 'source's ... so they are just renamed. We have 'derived' but it does not accept lists for now.  Basically any variable requiring sign changes would needto be explicitly specified (as 'derived') in a fixe